config/application.rb

```diff
@@ -13,13 +13,15 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
     config.autoload_paths += %W[
+      #{config.root}/app/models/imports
```

Comment:
i don't mess with config files much, but what does it do to `autoload` or `eagerload` these dirs?

Reply:
By default in development, when you reference something like `SomeModel` in Rails, it's going to go looking in places like `app/models/some_model.rb` (converting camel case to underscores).  And for something like `Blah::SomeModel` - it's going to look in places with a subfolder like `app/models/blah/some_model.rb`.

And if it doesn't find the file there, it throws an error.

This would be problematic for the `PodcastRssImporter` ... because i threw it in the imports directory without calling it `Imports::PodcastRssImporter`.  It's an aesthetic thing - I tend to like leaving the namespace out, but keeping them organized in a subfolder.  You'll notice other sub-models (like `Content` and `Uncut`) are in the top-level directory.